tensorflow/lite/python/interpreter.py

```diff
@@ -456,6 +456,16 @@ def invoke(self):
   def reset_all_variables(self):
     return self._interpreter.ResetVariableTensors()
 
+  def set_num_threads(self, i):
+    """Set number of threads used by TFLite kernels.
+
+    If not set, kernels are running single-threaded. Note that currently,
+    only some kernels, such as conv, are multithreaded.
+
+    Args:
+      i: number of threads.
+    """
+    return self._interpreter.SetNumThreads(i)
 
 class InterpreterWithCustomOps(Interpreter):
   """Interpreter interface for TensorFlow Lite Models that accepts custom ops.
```

Comment:
What about making the number of threads an argument to the constructor instead? We're trying to move away from making non-trivial properties mutable after construction.

Reply:
sure, will do it.

Reply:
Does creating a `Options` argument that contains a `numberOfThreads` option as what you did in Java binding [1] make sense to you? I mean to make it more general instead of number of threads specific.

[1] https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/java/src/main/java/org/tensorflow/lite/Interpreter.java#L233-L235

Reply:
Python has much better handling of named, selective arguments for constructors (and functions generally), so I don't think we need an Options construct here. I think we can just add as an other named argument.

Reply:
OK. I'll add a `num_threads` argument with default value 1.

Reply:
rebased, cleaned up a bit, and moved the number of threads setting to the constructor  of `Interpreter`